extensions/terminal-suggest/src/completions/cd.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+const cdSpec: Fig.Spec = {
+	name: 'cd',
+	description: 'Change the shell working directory',
+	args: {
+		name: 'folder',
+		template: 'folders',
+		isVariadic: true,
+
+		// Add an additional hidden suggestion so users can execute on it if they want to
+		suggestions: [
+			{
+				name: '-',
+				description: 'Switch to the last used folder',
+				hidden: true,
+			},
```

Comment:
Created https://github.com/microsoft/vscode/issues/234382 to tweak this for pwsh